CMakeLists.txt

```
@@ -3,4 +3,10 @@ cmake_minimum_required(VERSION 3.20)
 project("password_archive")
 
 file(GLOB srcs "src/**")
-add_executable(${PROJECT_NAME} ${srcs})
+add_executable(${PROJECT_NAME} ${srcs}
+        libs/sqlite/include/sqlite3.h
+        libs/sqlite/include/sqlite3ext.h)
+add_subdirectory(libs)
```

it's better to move add_subdirectory(libs) to make all libs be precompiled first before main app start compiling

```suggestion
add_subdirectory(libs)

file(GLOB srcs "src/**")
add_executable(${PROJECT_NAME} ${srcs}
        libs/sqlite/include/sqlite3.h
        libs/sqlite/include/sqlite3ext.h)
```